conans/test/functional/toolchains/cmake/test_cmake_toolchain_vxworks_clang.py

```diff
@@ -0,0 +1,62 @@
+import platform
+import textwrap
+
+import pytest
+
+from conans.test.assets.cmake import gen_cmakelists
+from conans.test.assets.sources import gen_function_cpp
+from conans.test.utils.tools import TestClient
+from conans.util.files import save
+
+
+@pytest.fixture
+def client():
+    c = TestClient()
+    save(c.cache.new_config_path, "tools.env.virtualenv:auto_use=True")
+    clang_profile = textwrap.dedent("""
+        [settings]
+        arch=armv7
+        build_type=RelWithDebInfo
+        compiler=clang
```

Comment:
Is ``clang`` the compiler used to cross-compile to ``vxworks``?

For cross compiling, it is typically necessary to provide a ``CC=full/path/to/cross/compiler`` env-var

Reply:
clang and ldarm are available in the path in my docker container.

Reply:
Ok, good, I was mostly wondering if ``clang`` is the compiler actually used to compile to vxworks, and no other compiler should go here.